docs/UpdatedData.md

```diff
@@ -0,0 +1,290 @@
+
+```json
+{
+  "time": "2020-05-12T08:50:08.000Z",
+  "timezoneOffset": 120,
+  "deviceTime": "2020-05-12T08:50:08",
+  "entryTime": "2020-05-12T08:45:08.000Z",
+  "deviceId": "IdOfTheDevice",
+  "type": "wizard",
+  "carbInput": 50,
+  "insulinOnBoard": 5.0,
+  "recommended": {
+    "net": 5
+  },
+  "units": "mg/dL",
+  "bolus": {
+    "time": "2020-05-12T08:50:08.000Z",
+    "timezoneOffset": 120,
+    "deviceTime": "2020-05-12T08:50:08",
+    "deviceId": "IdOfTheDevice",
+    "type": "bolus",
+    "subType": "normal",
+    "normal": 3.5,
+    "expectedNormal": 4.0, 
+    "prescriptor": "hybrid"
+  }
+}
+```
+
+## food 
+
+As of now we don't have the information of the origin of the rescueCarbs value, is it a patient decision, is it a system recommendation, and in that case what was the recommendation vs the actual value.
+
+Here we are introducing 2 new fields in the food object:
+- `prescribedNutrition`: same structure as nutrition. It's an optional field. It gives the value that has been recommended by the system. 
+- `prescriptor`: is the origin of the `rescuecarbs` object. This field is optional in most of the cases. 
+    - range of values: `auto | manual | hybrid`
+    - `auto`: nutrition and prescribedNutrition are equal
+    - `manual`: prescribedNutrition is ignored
+    - `hybrid`: nutrition and prescribedNutrition are __not equal__, `prescribedNutrition` is mandatory in that case. 
+
+```json
+{
+  "type": "food",
+  "meal": "rescuecarbs",
+  "nutrition": {
+    "carbohydrate" : {
+      "net": 20,
+      "units": "grams"
+    }
+  },
+  "prescribedNutrition": {
+    "carbohydrate" : {
+      "net": 30,
+      "units": "grams"
+    }
+  },
+  "prescriptor": "hybrid",
+  "meal": "rescuecarbs",
+  "deviceId": "IdOfTheDevice",
+  "deviceTime": "2020-05-12T06:50:08",
+  "time": "2020-05-12T06:50:08.000Z",
+  "timezoneOffset": 120
+}
+
+```
+
+## bolus
+
+3 types of bolus events are available as of now in the system:
+- normal
+- square
+- dual/square
+
+Here we are introducing 2 new fields in the bolus objects:
+- `prescriptor`: same as above in `food`. This field is optional. 
+- `insulinOnBoard`: amount of active insulin estimated by the system. This field will be accepted when `prescriptor` is either `auto` or `hybrid`. It will be ignored for `manual` entries.
+
+```json
+{
+  "time": "2020-05-12T08:50:08.000Z",
+  "timezoneOffset": 120,
+  "deviceTime": "2020-05-12T08:50:08",
+  "deviceId": "IdOfTheDevice",
+  "type": "bolus",
+  "subType": "normal",
+  "normal": 3.5,
+  "expectedNormal": 4.0, 
+  "prescriptor": "hybrid"
+}
+```
+
+## biphasic bolus
+
+A `biphasic` bolus is a 2 parts bolus that is defined by the system. Below is the definition for this new type of bolus that leverages most of the fields from `Normal` bolus. The subType associated to this type of bolus is `biphasic`.
+We add the following fields:
+- `bolusId`: unique ID provided by the client that is used to link the 2 parts of the bolus.
```

Comment:
We introduce the eventId field to link deviceEvents objects, and here we use bolusId field to link bolus objects. 

I'm not sure we need to create two different fields. The drawbacks I see : 
 - we will have to create two indices in the future if we have to optimize queries to get linked objects
 - data clients (i.e. blip for now) will have two processes to link objects together

Reply:
Good point. 
let's use the same field for linking objects.